Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -8693,6 +8749,28 @@ a_Player:OpenWindow(Window);
 					},
 					Notes = "Sets the custom name visiblity of this monster. If it's false, you only see the name when you sight the mob. If it's true, you always see the custom name.",
 				},
+				SetLeashedTo =
+				{
+					Params =
+					{
+						{
+							Name = "Entity",
+							Type = "cEntity",
+						},
+					},
+					Notes = "Leash the monster to a entity.",
+				},
+				SetUnleashed =
+				{
+					Params =
+					{
+						{
+							Name = "ShouldDropPickup",
+							Type = "boolean",
+						},
+					},
+					Notes = "Unleash the monster from an entity.",
```

Comment:
Sorry but I don't know Lua capabilities over Cuberite very well. So I don't have the knowledge to expose this functions in the right way. Maybe only cEntity.AddLeashedMob / cEntity.RemoveLeashedMob should be called from Lua plugins because there is all the work needed, including the protocol thing. In fact Monster.LeashTo and Monster.Unleash are called from there, so they are kind of internal functions.
So, can I add lua_begin / lua_end in cEntity in order to export those functions (cEntity.AddLeashedMob / cEntity.RemoveLeashedMob)? They are currently not being exported.